changelog.md

```diff
@@ -21,6 +21,7 @@
 - [#2981](https://github.com/ignite/cli/issues/2981) Change CLI to also search chain binary in Go binary path.
 - [#2958](https://github.com/ignite/cli/pull/2958) Support absolute paths for client code generation config paths.
 - [#2993](https://github.com/ignite/cli/pull/2993) Hide `ignite scaffold band` command and deprecate functionality.
+- [#2986](https://github.com/ignite/cli/issues/2986) Remove `--proto-all-modules` flag from `serve` command.
```

Comment:
Is it the only thing we should mention here ?

Reply:
🤔 well, good point, I will add more important details